src/Authentication.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pest\Laravel;
+
+use Illuminate\Contracts\Auth\Authenticatable;
+use Illuminate\Foundation\Testing\TestCase;
+
+/**
+ * Set the currently logged in user for the application.
+ *
+ * @return TestCase
+ */
+function actingAs(Authenticatable $user, string $driver = null)
+{
+    return test()->actingAs(...func_get_args());
```

Comment:
Why aren’t you using $user & $driver? 🤔 Also relevant for the methods below.

Reply:
To avoid mistakes, I am *always* using this approach `...func_get_args()`.